### DISCUSSION TITLE
刷新按钮能不能一致保持，而不是下拉之后就变成回到顶部了

### DISCUSSION THREAD

**Discussion by lalalallu** [BewlyBewly/BewlyBewly#814] [General]
刷新按钮能不能一致保持，而不是下拉之后就变成回到顶部了

**Comment by lalalallu** [comment]
![图片](https://github.com/BewlyBewly/BewlyBewly/assets/92630009/d93ea76b-06e4-4c00-b863-ce9c873f1271)
![图片](https://github.com/BewlyBewly/BewlyBewly/assets/92630009/26599a5d-fac8-4cc5-8f5a-3a091faf2d79)

**Reply by hakadao** [reply]
我覺得沒有必要多佔用一個位置

**Reply by lalalallu** [reply]
但是我往下拉，就不能刷新页面了啊

**Reply by hakadao** [reply]
回到頂部再刷新

**Reply by lalalallu** [reply]
那这不是很呆吗，为啥要做两次操作，还点击的是同一个按钮，不如把他们的功能分开呢

**Reply by hakadao** [reply]
普遍手機 app 的底部 tab bar 也是點擊第一下是回到頂部之後才是刷新數據, 我不覺得要調整

**Reply by lalalallu** [reply]
但这是网页啊，网页点击回到开头不是很没必要吗，能只点击一次为何要多次点击

**Reply by hakadao** [reply]
我覺得明明可以到底部再載入新數據的再加個刷新按鈕已經很弱智, 這個按鈕我甚至是爲了妥協才加的
